frontend/src/app/pages/Profile/index.js

```diff
@@ -1,30 +1,37 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux'
-import { 
+import {
   Title,
   Subtitle,
   Box, ProfileDetail
 } from './../../../styles/blocks';
 import ProfileBattleHistory from './../../components/ProfileBattleHistory'
+import ProfileRatingGraph from '../../components/ProfileRatingGraph'
 import { getUser } from '../../modules/users/users-selectors'
-import { getStatisticsForUser } from '../../modules/matches/matches-selectors';
+import { getStatisticsForUser } from '../../modules/matches/matches-selectors'
 
 class ProfileComponent extends Component {
   render() {
     if (!this.props.user) {
       return <div />
     }
 
-    return(
+    const {
+      user: { name, rating, id },
+      statistics: { totalMatches, winRatio, longestStreak, matchChanges }
+    } = this.props
```

Comment:
Some of the variables from the deconstruction are not used, right?

Reply:
Nope, all of them are used.